File: c_vs_java_vs_ruby_vs_python/twoloops_with_counter.c
#include <stdio.h>

int main() {
  int i = 0, j;
  long int counter = 0;
  while (i < 100000) {
    j = i;
    while (j < 100000) {
      j++;
      counter++;
    }
    i++;
  }
  printf("%ld iterations\n", counter);
}

File: c_vs_java_vs_ruby_vs_python/twoloops_with_counter.rs
fn main() {
    let mut i = 0;
    let mut j;
    let mut counter:u64 = 0;
    while i < 100000 {
        j = i;
        while j < 100000 {
            j += 1;
            counter += 1;
        }
        i += 1;
    }
    println!("{} iterations", counter);
}
